.github/workflows/build_dev_self_hosted.yml

```diff
@@ -173,8 +185,14 @@ jobs:
           echo MODULES_MODULE_NAME=$MODULES_MODULE_NAME
           echo MODULES_MODULE_SOURCE=$MODULES_MODULE_SOURCE
           echo MODULES_MODULE_TAG=$MODULES_MODULE_TAG
-        shell: bash
-
+      
+      - name: Cleanup build folder
+        run: |
+          ls -la ./
+          rm -rf ./* || true
+          rm -rf ./.??* || true
+          ls -la ./
```

Comment:
It's better to put this step at the end with `if: always()`

Reply:
```suggestion
```